diff --git a/node_modules/vuedraggable/dist/vuedraggable.common.js b/node_modules/vuedraggable/dist/vuedraggable.common.js
index 2350144..6364600 100644
--- a/node_modules/vuedraggable/dist/vuedraggable.common.js
+++ b/node_modules/vuedraggable/dist/vuedraggable.common.js
@@ -4511,10 +4511,13 @@ var componentStructure_ComponentStructure = /*#__PURE__*/function () {
       var defaultNodes = this.defaultNodes,
           realList = this.realList;
       defaultNodes.forEach(function (node, index) {
-        addContext(getHtmlElementFromNode(node), {
-          element: realList[index],
-          index: index
-        });
+        var el = getHtmlElementFromNode(node);
+        if (el) {
+          addContext(el, {
+            element: realList[index],
+            index: index
+          });
+        }
       });
     }
   }, {
diff --git a/node_modules/vuedraggable/dist/vuedraggable.umd.js b/node_modules/vuedraggable/dist/vuedraggable.umd.js
index 5a38c45..cb8b8d1 100644
--- a/node_modules/vuedraggable/dist/vuedraggable.umd.js
+++ b/node_modules/vuedraggable/dist/vuedraggable.umd.js
@@ -4520,10 +4520,13 @@ var componentStructure_ComponentStructure = /*#__PURE__*/function () {
       var defaultNodes = this.defaultNodes,
           realList = this.realList;
       defaultNodes.forEach(function (node, index) {
-        addContext(getHtmlElementFromNode(node), {
-          element: realList[index],
-          index: index
-        });
+        var el = getHtmlElementFromNode(node);
+        if (el) {
+          addContext(el, {
+            element: realList[index],
+            index: index
+          });
+        }
       });
     }
   }, {
diff --git a/node_modules/vuedraggable/src/core/componentStructure.js b/node_modules/vuedraggable/src/core/componentStructure.js
index e4a74ad..c9a8636 100644
--- a/node_modules/vuedraggable/src/core/componentStructure.js
+++ b/node_modules/vuedraggable/src/core/componentStructure.js
@@ -30,10 +30,13 @@ class ComponentStructure {
   updated() {
     const { defaultNodes, realList } = this;
     defaultNodes.forEach((node, index) => {
-      addContext(getHtmlElementFromNode(node), {
-        element: realList[index],
-        index
-      });
+      const el = getHtmlElementFromNode(node);
+      if (el) {
+        addContext(el, {
+          element: realList[index],
+          index
+        });
+      }
     });
   }
 
